sales/amazon_connector/setup.rst

```diff
@@ -2,16 +2,66 @@
 Configure Amazon Connector in Odoo
 ==================================
 
+The Amazon Connector uses an Authorization Token that allows the Odoo Amazon
+platform to fetch data from your Amazon Seller Central account. This token can
+be obtained directly through Seller Central in a few clicks and will then need
+to be set up in your Odoo database configuration.
+
+Generate an Authorization Token in Seller Central
+=================================================
+
+Connect to your Seller Central account for the marketplace you initially signed
+up (e.g. if you created your account on Amazon Germany, go to
+`Amazon Seller Central for Germany <https://sellercentral.amazon.de>`_) with an administrator account.
+
+.. note::
+  You might need to use the *main* (or first) administrator account and not one
+  added subsequentely.
+
+In the main menu, select :menuselection:`Apps & Services --> Manage Your Apps`;
+in the page that displays, click on the ``Authorize new developper`` button:
+
+.. image:: ./media/seller_central_apps.png
+  :align: center
+
+Fill in the form depending on your Marketplace:
+
+.. image:: ./media/seller_central_form.png
+  :align: center
+
+For Canada, Mexico or the United States, use these values:
+
+- Developer's Name: ``Odoo S.A.``
+- Developer ID: ``586127723692``
```

Comment:
@AntoineVDV plz confirm it's correct

Reply:
I hereby confirm the correctness of this information.

Reply:
![382909-20.jpeg](https://user-images.githubusercontent.com/9530767/88529591-7df4e400-d000-11ea-824a-90cfe4361b96.jpeg)